core/src/main/scala/pureconfig/CollectionReaders.scala

```diff
@@ -8,26 +8,26 @@ import pureconfig.ConvertHelpers._
 import pureconfig.error._
 
 /**
- * The default behavior of ConfigReaders that are implicitly derived in PureConfig is to raise a
- * KeyNotFoundException when a required key is missing. Mixing in this trait to a ConfigReader
- * allows customizing this behavior. When a key is missing, but the ConfigReader of the given
- * type extends this trait, the `from` method of the ConfigReader is called with null.
+ * A marker trait signaling that a `ConfigReader` accepts missing (undefined) values.
+ *
+ * The standard behavior of `ConfigReader`s that expect required keys in config objects is to return a `KeyNotFound`
+ * failure when one or more of them are missing. Mixing in this trait into the key's `ConfigReader` signals that a
```

Comment:
I think there's an `if` missing: `Mixing in this trait into the key's `ConfigReader` signals that if a value is missing...`.